vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/verdict.rb

```diff
@@ -0,0 +1,26 @@
+module SecureRooms
+
+  module AccessRules
+
+    class Verdict
```

Comment:
I still think I want these to separate classes, but I'm willing to punt on that until we get to some of the next rules and see if becomes necessary.